pkg/apis/apis/v1alpha1/types.go

```diff
@@ -339,6 +339,10 @@ type APIExportSpec struct {
 	//
 	// +optional
 	Identity *Identity `json:"identity"`
+
+	// policy will allow for a service provider to set a upper bound on what is allowed
```

Comment:
```suggestion
	// policy will allow for a service provider to set a upper bound on what is allowed for a consumer of this API.
```

Reply:
Wondering about a better field name. Something like "maximalPermissionPolicy" ?

Reply:
Changed the name, with a TODO to come back before beta.